std/jwt/signature.ts

```diff
@@ -0,0 +1,48 @@
+import type { Algorithm } from "./algorithm.ts";
+import { HmacSha256 } from "../hash/sha256.ts";
+import { HmacSha512 } from "../hash/sha512.ts";
+import { encode as convertUint8ArrayToBase64url } from "../encoding/base64url.ts";
+import { decodeString as convertHexToUint8Array } from "../encoding/hex.ts";
+
+export function convertHexToBase64url(input: string): string {
+  return convertUint8ArrayToBase64url(convertHexToUint8Array(input));
+}
+
+function encrypt(
+  alg: Algorithm | "none",
+  key: string,
+  message: string,
+): string {
+  switch (alg) {
+    case "none":
+      return "";
+    case "HS256":
+      return new HmacSha256(key).update(message).toString();
+    case "HS512":
+      return new HmacSha512(key).update(message).toString();
+    default:
+      throw new RangeError(`algorithm '${alg}' in header is not supported`);
+  }
+}
+
+export async function create(
```

Comment:
If functions exported from this module are part of the public API, then they should have JSDoc.  If they are just supporting `mod.ts` then the whole module should start with an underscore (`_`).